src/room.py

```diff
@@ -1,2 +1,49 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+# need room title and description
+
+
+class Room:
+    def __init__(self, name, description, items=[]):
+        self.name = name
+        self.description = description
+        self.items = items
+        self.n_to = None
+        self.e_to = None
+        self.s_to = None
+        self.w_to = None
+
+    def travel(self, direction):
```

Comment:
a couple things you might want to add to this function is `.lower()` and `.slice()` to handle the use cases of "N" or "North"... or just add a comment "Precondition - direction must be a single lowercase char of n, e, s, w" if these conditions are handled elsewhere in the code.